app/logic/events.py

```diff
@@ -277,6 +276,12 @@ def preprocessEventData(eventData):
         except DoesNotExist:
             eventData['term'] = ''
 
+    if 'startTime' in eventData:
+        eventData['startTime'] = format24HourTime(eventData['startTime'])
+
+    if 'endTime' in eventData:
+        eventData['endTime'] = format24HourTime(eventData['endTime'])
```

Comment:
I'm a little confused if we are passing around a string of 24 hour time everywhere, or a datetime object

Reply:
The code in the view assumes it is an object

Reply:
This function is expecting a string in a 12-hour format because this is where we are processing the data that comes from the user (js). However, we did fix the if statements because it is supposed to be 'timeStart' and 'timeEnd'.